lms/static/js/certificates/views/certificate_whitelist_editor.js

```diff
@@ -0,0 +1,87 @@
+// Backbone Application View: CertificateWhiteList Editor View
+/*global define, RequireJS */
+
+;(function(define){
+    'use strict';
+
+    define([
+            'jquery',
+            'underscore',
+            'gettext',
+            'backbone',
+            'js/certificates/models/certificate_exception'
+        ],
+
+        function($, _, gettext, Backbone, CertificateExceptionModel){
+            return Backbone.View.extend({
```

Comment:
I'm seeing a lot of unnecessary whitespace here -- can we tighten up the code a bit?